### PR TITLE
#CodeTheRoad blogpost

### PR DESCRIPTION
**Pull request by stefanhoth**
In order to support Google's CodeTheRoad event and thank them for their DevFest engagement we need to publish a blog post in which we not only mention their workshop and talk of the devfest but also link to https://lp.google-mkto.com/CodetheRoad-Berlin.html
### Result

image

### CODE REVIEW

File: _posts/2015-11-06-code-the-road.markdown [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**[Click here to register for the event!](https://lp.google-mkto.com/CodetheRoad-Berlin.html)**
+
+If you're asked for your favorite cookie please say "devfest" to let them know we helped you find it.
+
+---
+
+##### Google Maps @ DevFest Berlin
+
+Since the team is in town anyway they agreed to not only giving a talk on Google Maps APIs but instead Enoch will also give a workshop on the workshop day.
```
Comment (dlsniper) [inline _posts/2015-11-06-code-the-road.markdown]: I would rather say something along the line of:

```
The team will be in town not only to give a talk but Enoch, <link to speaker profile goes here>, will also give an awesome workshop in our workshop day <link to workshop>
```

Comment (stefanhoth) [inline _posts/2015-11-06-code-the-road.markdown]: How about:

`The team offered to not only give a talk on Google Maps APIs but [Enoch](/speakers/#speakerDetail-21) will also give a workshop on the workshop day.`

**Comment by dlsniper** [comment]
I'll merge this and bare the consequences.

**Comment by stefanhoth** [comment]
Thanks!

**Pull request merged**